init-tools.sh

```diff
@@ -47,7 +45,7 @@ if [ ! -e $__PROJECT_JSON_FILE ]; then
     echo "Running: $__scriptpath/init-tools.sh" > $__init_tools_log
     if [ ! -e $__DOTNET_PATH ]; then
         echo "Installing dotnet cli..."
-        __DOTNET_LOCATION="https://dotnetcli.blob.core.windows.net/dotnet/preview/Binaries/${__DOTNET_TOOLS_VERSION}/${__DOTNET_PKG}.${__DOTNET_TOOLS_VERSION}.tar.gz"
+        __DOTNET_LOCATION="https://corefxci.blob.core.windows.net/dotnetcli/${__DOTNET_PKG}.${__DOTNET_TOOLS_VERSION}.tar.gz"
```

Comment:
If you would like, I can make a similar change of the source in init-tools.cmd as well